x-pack/plugins/security_solution/server/fleet_integration/fleet_integration.test.ts

```diff
@@ -434,25 +446,51 @@ describe('ingest_integration tests ', () => {
     const soClient = savedObjectsClientMock.create();
     const esClient = elasticsearchServiceMock.createClusterClient().asInternalUser;
 
+    const infoResponse = {
+      cluster_name: 'updated-name',
+      cluster_uuid: 'updated-uuid',
+      name: 'name',
+      tagline: 'tagline',
+      version: {
+        number: '1.2.3',
+        lucene_version: '1.2.3',
+        build_date: 'DateString',
+        build_flavor: 'string',
+        build_hash: 'string',
+        build_snapshot: true,
+        build_type: 'string',
+        minimum_index_compatibility_version: '1.2.3',
+        minimum_wire_compatibility_version: '1.2.3',
+      },
+    };
+
+    esClient.info.mockResolvedValue(infoResponse);
+
     beforeEach(() => {
       licenseEmitter.next(Platinum); // set license level to platinum
     });
     it('updates successfully when meta fields differ from services', async () => {
       const mockPolicy = policyFactory();
       mockPolicy.meta.cloud = true; // cloud mock will return true
       mockPolicy.meta.license = 'platinum'; // license is set to emit platinum
+      mockPolicy.meta.cluster_name = 'updated-name';
```

Comment:
was the license_uuid intentionally left out of this test?

Reply:
I missed this, thanks for pointing it out!  I updated the test